package/MDAnalysis/analysis/psa.py

```diff
@@ -207,7 +251,85 @@ def get_path_metric_func(name):
         for name in path_metrics.keys(): print("  \"{}\"".format(name))
 
 
-def hausdorff(P,Q, N=None):
+def sqnorm(v, axis=None):
+    """Compute the sum of squares of elements along specified axes.
+
+    :Arguments:
+      *v*
+         :class:`numpy.ndarray` of coordinates
+      *axes*
+         None or int or tuple of ints, optional
+         Axes or axes along which a sum is performed. The default (*axes* =
+         ``None``) performs a sum over all the dimensions of the input array.
+         The value of *axes* may be negative, in which case it counts from the
+         last axis to the zeroth axis.
+
+    :Returns:
+      float, the sum of the squares of the elements of *v* along *axes*
+    """
+    return np.sum(v*v, axis=axis)
+
+
+def get_msd_matrix(P, Q, axis=None):
+    """Generate the matrix of pairwise mean-squared deviations (MSDs) between
+    all pairs of points in *P* and *Q*, each pair having a point from *P* and a
+    point from *Q*.
+
+    *P* (*Q*) is a :class:`numpy.ndarray` of :math:`N_p` (:math:`N_q`) time
+    steps, :math:`N` atoms, and :math:`3N` coordinates (e.g.,
+    :meth:`MDAnalysis.core.AtomGroup.AtomGroup.coordinates`). The pairwise MSD
+    matrix has dimensions :math:`N_p` by :math:`N_q`.
+
+    :Arguments:
```

Comment:
It's pretty cruel to make him rewrite them, I'm happy to use either, maybe just make it consistent within a submodule